utils/wui.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+
+import os
+from pathlib import Path
+import gzip
+import shutil
+
+class WUIFiles:
+    def __init__(self):
+        self.wui_files = []
+        # path to resource files directory
+        self.wui_path = Path(__file__).resolve().parent.parent / 'lib' / 'WUI' / 'resources' / 'src_local'
+        # path to main C file with raw data
+        self.main_file_name = 'fsdata_wui_local.c'
+        self.raw_data_file = Path(__file__).resolve().parent.parent / 'lib' / 'WUI' / 'resources' / self.main_file_name
+        self.raw_data_file_tmp = Path(__file__).resolve().parent.parent / 'lib' / 'WUI' / 'resources' / str(self.main_file_name + '_tmp')
+
+        # start
+        self.getFiles()
+        self.gzipFiles()
+        self.generateRawDataFile()
+    #endef
+
+    # store paths to static wui files in array for later use
+    def getFiles(self):
```

Comment:
/suggestion/ Let's please follow the so-far only naming convention we have for Python in our repository and change the method names to snake_case

Reply:
I will change that - sorry